src/aleph/http/core.clj

```diff
@@ -544,7 +553,7 @@
                 (try
                   (send-streaming-body ch msg body)
                   (catch Throwable e
-                    (log/error e "error sending body of type " class-name)
+                    (log/error "error sending body of type " class-name)
```

Comment:
Since the exception is thrown to the client, we don't need to log anymore.

Reply:
Why not? Client and server could be different parties, yes?

Reply:
312ffa88ad488c10ea26009dee8234e79585da9f

Reply:
Yes... reverted.